frontend/packages/kubevirt-plugin/src/utils/validations/template/template-validations.ts

```diff
@@ -110,6 +111,32 @@ export class TemplateValidations {
     });
   };
 
+  getAllowedBusses = (): Set<DiskBus> => {
+    const allowedBusses = new Set(
```

Comment:
https://github.com/openshift/console/pull/4028/files#r369505305, so you don't have to do it here before the return